Detections/AuditLogs/GuestAccountsAddedinAADGroupsOtherThanTheOnesSpecified.yaml

```diff
@@ -0,0 +1,53 @@
+id: 1caa690e-4d5c-4083-ba1a-97cdebad4eff
+name: Guest accounts added in AAD Groups other than the ones specified
+description: |
+  'Guest Accounts are added in the Organization Tenants to perform various tasks i.e projects execution, support etc.. This detection notifies when guest users are added to Azure AD Groups other than the ones specified and poses a risk to gain access to sensitive apps or data.'
+severity: High
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+queryFrequency: 5m
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+status: Available
+tactics:
+  - InitialAccess
+  - Persistence
+  - Discovery
+relevantTechniques:
+  - T1078.004
+  - T1136.003
+  - T1087.004
+query: |
+  let GroupIDs = dynamic(["List with Custom AAD GROUP OBJECT ID 1","Custom AAD GROUP OBJECT ID 2"]);
```

Comment:
Is it possible to provide a comment that suggests a way to find AAD group Object Id. 
That will be helpful for reference and hopefully, will also help customers to be remind this needs to be put in when deploying the template.

Reply:
added where to find objectid of AADGRoups